### ISSUE TITLE
Runtime error when using crop pooling

### ISSUE THREAD

**Issue by LowBoonKeong** [ruotianluo/pytorch-faster-rcnn#52]
Hi @ruotianluo , I meet the following runtime error when using crop pooling in CPU on vgg16 net. 
![12](https://user-images.githubusercontent.com/30174905/34070153-2160e234-e29b-11e7-8360-a6b41358e52d.PNG)
I hope to get some solution. Thank you

**Comment by ruotianluo** [comment]
What version of torch?

**Comment by LowBoonKeong** [comment]
I use the latest master branch update pytorch https://github.com/pytorch/pytorch by building it from source. Is it a pytorch version problem ?

**Comment by ruotianluo** [comment]
Try replace https://github.com/ruotianluo/pytorch-faster-rcnn/blob/master/lib/nets/network.py#L107
    x1 = rois[:, 1::4] / 16.0
    y1 = rois[:, 2::4] / 16.0
    x2 = rois[:, 3::4] / 16.0
    y2 = rois[:, 4::4] / 16.0
to
    x1 = rois[:, 1] / 16.0
    y1 = rois[:, 2] / 16.0
    x2 = rois[:, 3] / 16.0
    y2 = rois[:, 4] / 16.0

See if it helps.

**Comment by LowBoonKeong** [comment]
Hi @ruotianluo, the problem solved after change the above code. However, may I get some solution or idea about the following warning. 
![13](https://user-images.githubusercontent.com/30174905/34076282-d05fc4a0-e31b-11e7-8ac1-4f057b9ee79d.PNG) 
Thank you.

**Comment by leviswind** [comment]
@LowBoonKeong , I think it means you need to explicitly choose the dimensition, the original softmax treat the last dimension as the default choice, but now it will give you a warning.

**Comment by LowBoonKeong** [comment]
Hi, @leviswind , thanks very much for your help. Really appreciate it. The warning is really solved by explicitly specify a dimension for softmax.
